pallets/elections/src/lib.rs

```diff
@@ -122,6 +122,28 @@ pub mod pallet {
                     }
                 }
         }
+        #[cfg(feature = "try-runtime")]
+        fn pre_upgrade() -> Result<(), &'static str> {
+            let on_chain = <Pallet<T> as GetStorageVersion>::on_chain_storage_version();
+            match on_chain {
+                _ if on_chain == STORAGE_VERSION => Ok(()),
+                _ if on_chain == StorageVersion::new(0) => {
+                    migrations::v0_to_v1::pre_upgrade::<T, Self>()
```

Comment:
does it make sense to pass here the target version?

Reply:
wdym?

Reply:
to make this check aware of the fact, that finally we are updating to even higher version; but I guess this is completely unneeded